source/features/confirm-too-short-comments.tsx

```diff
@@ -0,0 +1,47 @@
+// import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+
+// function handleComment() {
+function handleComment(event: delegate.Event<MouseEvent, KeyboardEvent, HTMLAnchorElement>): void {
```

Comment:
Can you try merging these 2 functions? They do the same thing. You can merge the selectors in a single call for the textareas and then another check + question for the title, if present

Reply:
I merged the selectors together. 
I also combined the checks and question, all into one. Isn't this the same?

Reply:
If 2 versions are logically the same, one should prefer the shorter one 😉

Reply:
Thank you. 
I made the checks and questions separate for comment and for new issue.